coverage.py

```diff
@@ -61,19 +61,24 @@ def log(self):
         return self._log        
 
     @property
-    def editions_that_need_coverage(self):
-        """Find all Editions lacking coverage from this CoverageProvider.
+    def items_that_need_coverage(self):
```

Comment:
A CoverageProvider that operates on Editions should still work, but we don't use 'editions' anymore in method names and by default we operate on Identifiers.